pkg/serverless/daemon/daemon.go

```diff
@@ -390,6 +390,7 @@ func (d *Daemon) ComputeGlobalTags(configTags []string) {
 	if len(d.ExtraTags.Tags) == 0 {
 		ecs := d.ExecutionContext.GetCurrentState()
 		tagMap := tags.BuildTagMap(ecs.ARN, configTags)
+		d.ExecutionContext.UpdateRuntime(tagMap[tags.RuntimeKey])
```

Comment:
Do we need to add this tag to every span? i.e. do we need it to be global? I thought your goal was to add it to the aws.lambda span but its being added to every span since its global. In my mind that means we are duplicating data on the other spans, unless they serve a different purpose that i'm not aware of. I think we can do with just adding it to the aws.lambda span. https://github.com/DataDog/datadog-agent/pull/18357/files#diff-fa012404d0d0ddcd56f7d87ec2867b3d494c4a3ba620238ac7234bbee7329b29R115

Reply:
This line is not setting the tag. It's extracting the runtime information and put it into the ExecutionContext, so that later we can use the runtime information to set the tag [here](https://github.com/DataDog/datadog-agent/blob/239abc68dda31f14f1909a2dddc3aed65fbf1d84/pkg/serverless/invocationlifecycle/trace.go#L122-L124)

Reply:
ahh you're right, apologies